modules/plugins/utility/motion/flash/default.nix

```
@@ -0,0 +1,6 @@
+_: {
```

```suggestion
{
```

No need for an empty arg in files unlikely to get args.